### ISSUE TITLE
serializer_legacy tests depend on pymodaq_data

### ISSUE THREAD

**Issue by malik-irain** [PyMoDAQ/pymodaq_utils#7]
PR  #5  introduced a dependency to pymodaq_data in the test suite:

#### **`tests/serialize/serializer_legacy_test.py`**
```python
...
from pymodaq_data import data as data_mod
from pymodaq_data.data import Axis, DataToExport, DwaType
...
``` 

So they fail if exectuted in an environment where it is not available (github actions for example) or if one were to run tests while updating from pymodaq_utils and not (yet) data.

I feel like serialization / deserialization of Axis, Dwa, ... (stuff from pymodaq_data) should be tested in pymodaq_data and pymodaq_utils shouldn't even know about these types.

**Comment by seb5g** [comment]
yes you are right and good to see that ;-) As you can see this guy is called legacy... which is a first time serializer that should know every data it will handle. The new serializer is based on a factory pattern. New serializable object are added on the fly by a registration mechanism. @BenediktBurger is in the process of using such new serializer for tcp/ip communication. After his work, we will remove from v5 the old mechanisms. legacy will disapear.

For the time being, we should solve your issue by just adding a pytest tag (don't remember which one right now) to not test the legacy stuff (we know they are working righ now). Could you propose such a PR?

**Comment by malik-irain** [comment]
It should be included in PR #11
